examples/config.yaml

```diff
@@ -28,6 +28,9 @@ checks:
   hostPortSet: warning
   tlsSettingsMissing: warning
 
+mutations:
+  - pullPolicyNotAlways
```

Comment:
let's remove from the default config for now, so it's not enabled for everyone yet